src/app/notification-view-list/notification-view-list.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component, OnInit } from '@angular/core';
+import { UserService } from '../shared/user.service';
+import { CouchService } from '../shared/couchdb.service';
+
+@Component({
+  template: `
+    <p i18n>Your Notifications</p>
+    <mat-list role="list" *ngFor="let notification of notifications">
+      <mat-list-item (click)="readNotification(notification)">
+      <mat-divider></mat-divider>
+        <p [ngClass]="{'menu-item-text':notification.status==='unread'}">
+          <a routerLink="{{notification.link}}">
+            {{notification.message}} {{notification.time | date: 'MMM d, yyyy'}}
+          </a>
+        </p>
+      </mat-list-item>
+    </mat-list>
+  `,
+  styles: [ `
+    .menu-item-text {
```

Comment:
Please remove this style from this file and `home.scss`.

Add this style to `style.scss` below the `.accent-color` definition and use it instead of `menu-item-text` in the two locations:
```
.primary-text-color {
  color: $primary;
}
```